pkg/antctl/command_list_test.go

```diff
@@ -54,3 +63,172 @@ func TestCommandListApplyToCommand(t *testing.T) {
 	assert.Contains(t, testRoot.Short, fmt.Sprintf("The component is %s", runtime.Mode))
 	assert.Contains(t, testRoot.Long, fmt.Sprintf("The component is %s", runtime.Mode))
 }
+
+var testCommandList2 = &commandList{
```

Comment:
is it different from `CommandList` in antctl.go? If not, why making a copy here?

Reply:
made changes and used CommandList in antctl.go itself